.github/workflows/pytorch-linux-xenial-py3.6-gcc5.4.yml

```diff
@@ -122,7 +122,7 @@ jobs:
       - name: Chown workspace
         run: |
           # Ensure the working directory gets chowned back to the current user
-          docker run --rm -v "$(pwd)":/v -w /v alpine chown -R "$(id -u):$(id -g)" .
+          docker run --rm -v "$(pwd)/../":/v -w /v alpine chown -R "$(id -u):$(id -g)" .
```

Comment:
just wondering: what's the reason for mounting `pwd` instead of `docker cp`-ing the artifacts out?

Reply:
Volume mounting doesn't require an already running container, so essentially you're taking a bunch of commands like:
```
docker run -d --name dev alpine
docker cp $(pwd) dev:/v
docker exec -it chown -R "$(id -u):$(id -g) /v
docker cp dev:/v $(pwd)
```

And that condenses it down to one command:
```
docker run --name dev -v $(pwd):/v alpine chown -R $(id -u):$(id -g) /v
```

You could definitely do it with the multiple, but I tend to just prefer to do it with the one